apps/storefront/src/pages/ShoppingListDetails/components/ShoppingDetailFooter.tsx

```diff
@@ -253,7 +253,7 @@ function ShoppingDetailFooter(props: ShoppingDetailFooterProps) {
         const deleteCartObject = deleteCartData(cartEntityId);
         const cartInfo = await getCart();
         let res = null;
-        if (allowJuniorPlaceOrder && cartInfo.length) {
+        if (allowJuniorPlaceOrder && cartInfo?.data?.site?.cart) {
```

Comment:
I'm a bit concerned with this change, the original code has been broken for a while now.